templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/MenuGroups.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <b-container>
+    <div v-for="(menu, index) in filteredMenuGroups" :key="index" class="py-1">
+      <b-card>
+        <b-card-text
+          v-b-toggle="index === 0 || menu.length === 0 ? `` : `collapse-${index}`"

```

Comment:
If you change this component to a single menu group, you can easily pass this as a prop so this component doesn't have to deal with that responsibility.